CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
   - Fix url trailing slash and double slash issues (#659)
   - Fix accept all Content-Type by default (#653)
   - Return the error message from Serde when a deserialization error is encountered (#661)
+  - Fix set dashboard accessible (#695)
```

Comment:
What does this mean?
```suggestion
  - Make the dashboard accessible when a master key is used (#695)
```

Reply:
if I change it to `Fix NormalizePath middleware to set dashboard accessible`? I changed the NormalizePath middleware because the last one always remove the last `/` and for the dashboard route, that is only a `/`, the router never found it.